packages/components/Shadow/package.json

```diff
@@ -0,0 +1,45 @@
+{
+  "name": "@fluentui-react-native/shadow",
+  "version": "0.0.1",
+  "description": "A cross-platform Shadow component using the Fluent Design System",
+  "main": "src/index.ts",
+  "module": "src/index.ts",
+  "typings": "lib/index.d.ts",
+  "onPublish": {
+    "main": "lib-commonjs/index.js",
+    "module": "lib/index.js"
+  },
+  "scripts": {
+    "build": "fluentui-scripts build",
+    "clean": "fluentui-scripts clean",
+    "depcheck": "fluentui-scripts depcheck",
+    "just": "fluentui-scripts",
+    "lint": "fluentui-scripts eslint",
+    "test": "fluentui-scripts jest",
+    "update-snapshots": "fluentui-scripts jest -u",
+    "prettier": "fluentui-scripts prettier",
+    "prettier-fix": "fluentui-scripts prettier --fix true"
+  },
+  "repository": {
+    "type": "git",
+    "url": "https://github.com/microsoft/fluentui-react-native.git",
+    "directory": "packages/components/Shadow"
+  },
+  "dependencies": {
+    "@fluentui-react-native/experimental-text": ">=0.7.17 <1.0.0",
+    "@fluentui-react-native/framework": "0.7.17",
+    "@fluentui-react-native/tokens": ">=0.11.8 <1.0.0",
+    "@fluentui-react-native/use-styling": ">=0.8.2 <1.0.0"
+  },
+  "devDependencies": {
+    "@fluentui-react-native/eslint-config-rules": "^0.1.1",
+    "@fluentui-react-native/test-tools": ">=0.1.1 <1.0.0",
+    "@fluentui-react-native/scripts": "^0.1.1"
+  },
```

Comment:
Make sure these are all their latest, otherwise stuf gets added to the yarn.lock file

Reply:
How do I make sure these are all at their latest? Is there a command I should run or do I need to make changes manually?

Reply:
Ended up just removing the dependencies